apps/frontend/src/renderer/components/terminal/WorktreeSelector.tsx

```
@@ -0,0 +1,133 @@
+  // This component only shows when there's no worktree attached
+
+  return (
+    <DropdownMenu open={isOpen} onOpenChange={setIsOpen}>
+      <DropdownMenuTrigger asChild>
+        <button
+          className={cn(
+            'flex items-center gap-1 h-6 px-2 rounded text-xs font-medium transition-colors',
+            'hover:bg-amber-500/10 hover:text-amber-500 text-muted-foreground'
+          )}
+          onClick={(e) => e.stopPropagation()}
+        >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)